src/platform/Kconfig

```diff
@@ -193,6 +193,14 @@ config CAVS_VERSION_2_5
 	help
 	  Select for CAVS version 2.5
 
+config CHERRYTRAIL_DMA
```

Comment:
also check patch said I needed to add a paragraph, I did understand exactly what it wanted since it followed the same format as the other configs so I bypassed it